### PR TITLE
Allow pipelines to define deployment tag pattern

### PR DESCRIPTION
**Pull request by hammady**
Because rayyan-cloud repo hosts 2 charts, tagging for each chart is different. Consequently, the deployment tag pattern condition should accept parameters.

### CODE REVIEW

File: helm/scaltainer/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: scaltainer
 description: A Helm chart for scaltainer
 type: application
-version: 1.0.0
```
Comment (fabubaker) [inline helm/scaltainer/Chart.yaml:5]: Is the bump in chart version due to the change in the pipelines file?

Comment (hammady) [reply helm/scaltainer/Chart.yaml:5]: Yes, because I intend to create a new tag to deploy instead of deleting the old one.

Comment (fabubaker) [reply helm/scaltainer/Chart.yaml:5]: Should tags match the chart version or the app version? I thought it was the latter.

Comment (hammady) [reply helm/scaltainer/Chart.yaml:5]: Oh right, tags should match the `appVersion`.

Comment (hammady) [reply helm/scaltainer/Chart.yaml]: Fixed.

**Review by fabubaker — APPROVED** [review]

**Pull request merged**